### PR TITLE
Markdown linting fixes

### PR DESCRIPTION
**Pull request by 06kellyjac**
Should check none of the markdown changes broke anything rendered from the markdown

### CODE REVIEW

**Comment by sublimino** [comment]
LGTM, thanks @06kellyjac :+1:

File: README.md [hunk 1]
```diff
@@ -65,7 +64,7 @@ metadata:
   name: kubesec-demo
 spec:
   containers:
-  - name: kubesec-demo
+  * name: kubesec-demo
```
Comment (sublimino) [inline README.md]: This is YAML and should remain a `-` please :pray:

Comment (06kellyjac) [reply README.md]: oops.

In response to your preference for dashes, I can swap it all back if you prefer but then w have to  either change the CoC to use Dashes or ignore it from linting

**Comment by 06kellyjac** [comment]
I've used my `markdownlint-cli` container from my repo's GitLab docker registry. The docker hub images have been building for 2h. :weary: 
I've also made sure the `markdownlint-cli` image is using a non-root user & is all up to date

**Comment by sublimino** [comment]
Nice one @06kellyjac, thank you! :man_dancing:

**Pull request merged**